server/src/routes/api/v1/auth/index.ts

```diff
@@ -0,0 +1,88 @@
+import { Router, Request, Response, NextFunction } from "express";
```

Comment:
What do you think about renaming index.ts. I'm not sure why it's called index if it contains the routes. So with recipes I called this recipe.routes.ts.